packages/insured-bridge-relayer/src/LogHelper.ts

```diff
@@ -0,0 +1,44 @@
+import fs from "fs";
+import path from "path";
+import { Datastore } from "@google-cloud/datastore";
+const datastore = new Datastore();
+
+enum LogHelperMode {
+  Gcp,
+  Local,
+}
+
+const logHelperMode = process.env.LOG_SILENCER_MODE == "gcp" ? LogHelperMode.Gcp : LogHelperMode.Local;
+
+const localSaveFilePath = `${path.resolve(__dirname)}/relayer-unprofitable-logs`;
```

Comment:
What's the point of `path.resolve` here? Isn't __dirname already an absolute path?